lib/menu.rb

```diff
@@ -0,0 +1,32 @@
+# => Knows everything about menu
```

Comment:
circular - menu understands menu? Doesn't mean anything. Maybe the Menu understands the available dishes?